applications/convection_diffusion_application/python_scripts/convection_diffusion_base_solver.py

```diff
@@ -1,642 +1,642 @@
-        builder_and_solver = self.get_builder_and_solver()
-        return KratosMultiphysics.ResidualBasedLinearStrategy(computing_model_part,
-                                                              convection_diffusion_scheme,
-                                                              linear_solver,
-                                                              builder_and_solver,
-                                                              self.settings["compute_reactions"].GetBool(),
-                                                              self.settings["reform_dofs_at_each_step"].GetBool(),
-                                                              False,
-                                                              self.settings["move_mesh_flag"].GetBool())
-
-    def _create_newton_raphson_strategy(self):
-        computing_model_part = self.GetComputingModelPart()
-        convection_diffusion_scheme = self.get_solution_scheme()
-        linear_solver = self.get_linear_solver()
-        convection_diffusion_convergence_criterion = self.get_convergence_criterion()
-        builder_and_solver = self.get_builder_and_solver()
-        return KratosMultiphysics.ResidualBasedNewtonRaphsonStrategy(computing_model_part,
-                                        convection_diffusion_scheme,
-                                        linear_solver,
-                                        convection_diffusion_convergence_criterion,
-                                        builder_and_solver,
-                                        self.settings["max_iteration"].GetInt(),
-                                        self.settings["compute_reactions"].GetBool(),
-                                        self.settings["reform_dofs_at_each_step"].GetBool(),
-                                        self.settings["move_mesh_flag"].GetBool())
-
-    def _create_line_search_strategy(self):
-        computing_model_part = self.GetComputingModelPart()
-        convection_diffusion_scheme = self.get_solution_scheme()
-        linear_solver = self.get_linear_solver()
-        convection_diffusion_convergence_criterion = self.get_convergence_criterion()
-        builder_and_solver = self.get_builder_and_solver()
-        return KratosMultiphysics.LineSearchStrategy(computing_model_part,
-                            convection_diffusion_scheme,
-                            linear_solver,
-                            convection_diffusion_convergence_criterion,
-                            builder_and_solver,
-                            self.settings["max_iteration"].GetInt(),
-                            self.settings["compute_reactions"].GetBool(),
-                            self.settings["reform_dofs_at_each_step"].GetBool(),
-                            self.settings["move_mesh_flag"].GetBool())
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7

```

Comment:
please remove the files in the ConDiffApp, they are leftover from before the renaming of the folder